extra-web/pidgin-kwallet/autobuild/build

```diff
@@ -0,0 +1,5 @@
+mkdir -p "${SRCDIR}/PidginKWallet-${PKGVER}/build"
```

Comment:
This package is incomplete. And as this PR is already messy enough, I will close this PR until you open up a new one - on a new branch based off `staging` or a completely new Git clone, up to you.